meson.build

```diff
@@ -122,6 +122,7 @@ pkgconfiglibdir = get_option('pkgconfiglibdir') == '' ? join_paths(libdir, 'pkgc
 polkitpolicydir = join_paths(datadir, 'polkit-1/actions')
 polkitrulesdir = join_paths(datadir, 'polkit-1/rules.d')
 polkitpkladir = join_paths(localstatedir, 'lib/polkit-1/localauthority/10-vendor.d')
+firewalldzonesdir = join_paths(prefixdir, 'lib/firewalld/zones')
```

Comment:
i think this is something we might want to ship in the systemd RPM. But upstream? I am not convinced firewalld is conceptually or in code really something that makes so much sense. I really am a doubter on this one.

@keszybz what says you?

If fedora decided to adopt this, then fine, we can include it in the RPM, but upstream? that feels too much like an official blessing, and there's little I'd like to bless less than this...